.github/workflows/articles.yml

```diff
@@ -0,0 +1,17 @@
+name: <NAME> on Medium
```

Comment:
Why do we need this? We don not have a blog that supports RSS for now, and if it were, I definitely wouldn't add them to a Action's github readme